kafka-questdb-connector-samples/stocks/docker-compose.yml

```
@@ -1,13 +1,12 @@
-version: '2.1'
+version: "2.1"
 services:
   questdb:
-    image: questdb/questdb:6.5.4
+    image: questdb/questdb:6.6.1
     expose:
       - "9009"
     ports:
       - "19000:9000"
```

I wonder if this should be:
```suggestion
      - "9000:9000"
```
I can see why we may want to define a different address as a local instance may be using 9000 but I thought I'd ask!
The correct blog article draft is aligning with `19000`.

---

you are exactly right. I assume many users will have a local questdb already running and this is meant to avoid a port conflict.